packages/select/src/stories/Select.tw.stories.ts

```diff
@@ -0,0 +1,61 @@
+import { action } from '@storybook/addon-actions'
```

Comment:
this is autogenerated, if you don't plan to apply tailwind soon I recommend you remove it